schema/dsrc/dsrc_schema_2-0-0.json

```diff
@@ -0,0 +1,400 @@
+      "enum": [
+        "reserved",
+        "doNotBlock",
+        "whiteLine",
+        "mergingLaneLeft",
+        "mergingLaneRight",
+        "curbOnLeft",
+        "curbOnRight",
+        "loadingZoneOnLeft",
+        "loadingZoneOnRight",
+        "turnOutPointOnLeft",
+        "turnOutPointOnRight",
+        "adjacentParkingOnLeft",
+        "adjacentParkingOnRight",
+        "adjacentBikeLaneOnLeft",
+        "adjacentBikeLaneOnRight",
+        "sharedBikeLane",
+        "bikeBoxInFront",
+        "transitStopOnLeft",
+        "transitStopOnRight",
+        "transitStopInLane",
+        "sharedWithTrackedVehicle",
+        "safeIsland",
+        "lowCurbsPresent",
+        "rumbleStripPresent",
+        "audibleSignalingPresent",
+        "adaptiveTimingPresent",
+        "rfSignalRequestPresent",
+        "partialCurbIntrusion",
+        "taperToLeft",
+        "taperToRight",
+        "taperToCenterLine",
+        "parallelParking",
+        "headInParking",
+        "freeParking",
+        "timeRestrictionsOnParking",
+        "costToPark",
+        "midBlockCurbPresent",
+        "unEvenPavementPresent"
+      ]
+    },
+    "node_attribute_xy": {
+      "type": "string",
+      "description": "Enumerated list of attributes which can pertain to the current node point. The scope of these values is limited to the node itself. That is, unlike other types of attributes which can be switched on or off at any given node (and hence pertains to one or more segments), the DE_NodeAttribute is local to the node in which it is found. These attributes are all binary flags in that they do not need to convey any additional data. Other attributes allow sending short data values to reflect a setting which is set and persists in a similar fashion.",
+      "enum": [
+        "reserved",
+        "stopLine",
+        "roundedCapStyleA",
+        "roundedCapStyleB",
+        "mergePoint",
+        "divergePoint",
+        "downstreamStopLine",
+        "downstreamStartNode",
+        "closedToTraffic",
+        "safeIsland",
+        "curbPresentAtStepOff",
+        "hydrantPresent"
+      ]
+    },
+    "speed_limit_list": {
+      "type": "array",
+      "description": "List of SpeedLimit entries.",
+      "minItems": 1,
+      "maxItems": 9,
+      "items": {
+        "type": "object",
+        "required": [
+          "limit_type",
+          "speed"
+        ],
+        "properties": {
+          "type": {
```

Comment:
You named the property `type`, but in the required list, it is `limit_type`. The name is DSRC is indeed `type`.

Reply:
Fixed.